src/NUnitEngine/nunit.engine.tests/Runners/MasterTestRunnerTests.cs

```diff
@@ -105,13 +176,20 @@ public void Load()
             var result = _runner.Load();
 
             Assert.That(result.Name, Is.EqualTo("test-run"));
-            Assert.That(result.GetAttribute("testcasecount", 0), Is.EqualTo(MockAssembly.Tests * _numAssemblies));
+            Assert.That(result.GetAttribute("testcasecount", 0), Is.EqualTo(_totalTests));
```

Comment:
Picky again: zero is a valid value here now we're testing no-tests-assembly, right? Would it be a little stronger to default to `-1` for `GetAttribute()`? Same occurs in several other places in this file.